data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+				decline
+
+mission "Bactrian Unlock [0]: Pirates"
+	invisible
+	landing
+	destination Valhalla
+	to offer
+		has "Bactrian Unlock [0]: active"
+	to fail
+		or
+			has "Bactrian Unlock [3]: done"
+			has "Bactrian Unlock [3-alt]: offered"
+	npc kill
+		personality nemesis staying
+		system Arneb
+		government Pirate
+		ship "Leviathan (Laser)" "His Right Hand"
+		ship "Leviathan (Laser)" "His Left Hand"
+
+mission "Bactrian Unlock [1]"
+	name "Secure the Scientists"
+	description "Now that you have located the Scientists and their Star Queen, return to <destination> with the Deep escorts and retrieve it."
+	landing
+	destination Haven
+	stopover Memory
+	stopover Farpoint
+	clearance
+	infiltrating
+	to offer
+		has "Bactrian Unlock [0]: done"
+	on offer
+		conversation
+			`You return to <planet> to find the man that you met earlier in the spaceport along with a large group of people in Deep Security uniforms.`
+			`	As you approach them, the man stops talking with someone and notices you. "Captain <last>! Have you located the Star Queen?"`
+			choice
+				`	"Yes, it is located on <destination>, but is heavily defended."`
+			
+			branch destroyed
+				has "Bactrian Unlock [0]: Pirates: done"
+			`	You mention the multiple heavy warships that you saw over the planet, and the fact that there are probably many more waiting on the planet.`
+				goto next
+			
+			label destroyed
+			`	You mention the multiple heavy warships that you took out over the planet, but that there may be many more waiting on the planet.`
+			
+			label next
+			`	After hearing this news, many of the faces in the group turn grim as they realize that they will need to fight to retrieve the Star Queen. `
+			`	"We will need to get a rescue fleet ready. Soldiers, ready your ships!" The group responds with a collective "Sir yes sir!" before running off to their ships.`
+			`	Before leaving for his own ship, the man walks up to you and hands you 100,000 credits.`
+			`	"That's for finding the Star Queen. Now, Captain <last>, would you be willing to help us? If the Star Queen is as heavily defended as you say it is then we're going to need every ship we can get."`
+			choice
+				`	"I'd be glad to."`
+				`	"I would rather let the professionals handle things. Good luck."`
+					decline
+			
+			`	"Thank you so much. We'll be rendezvousing with a number of warships on Memory, and I've also sent word to the Navy on Farpoint that we may be needing help. After that we'll head out to Arneb, but just make sure that the D.S.S. Faraday Corvette makes it onto the planet. The landing crew will be on that ship. Just launch whenever you're ready to fight."`
+			`	The man starts talking into some communication device, giving orders to the fleet that will be joining you on the mission, then puts the device back on his wasit and looks to you. "Also, since we'll be going into battle together, I might as well introduce myself. I'm Deep Lieutenant Shane Paris. Good luck fighting."`
```

Comment:
"then puts the device back on his wasit" what's a wasit? and whatever it is, shouldn't he put it IN his wasit?